manifest.json

```diff
@@ -1,29 +1,35 @@
 {
   "name": "SQL Importer",
-  "description": "Import SQL data from your own database",
+  "description": "Import data from your SQL databases",
   "picture": "picture.png",
-  "version": "0.2.3",
+  "version": "0.3.0",
```

Comment:
We try to stick with Git Flow: http://nvie.com/img/git-model@2x.png

In that flow when doing a PR to develop you should not need to think about the versioning. After we decide that develop should go to production we create new release branch and mark new version.